portafly/package.json

```diff
@@ -88,10 +88,10 @@
     "ts-pnp": "1.2.0",
     "typescript": "~3.8.3",
     "url-loader": "4.1.0",
-    "webpack": "4.43.0",
```

Comment:
Why it's needed to downgrade this packages?

Reply:
The config file supports the old version, it's better to be aligned with CRA releases.

In other words, our config doesn't work with 4.43